IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_AssetManagementInformation/DocPropertySet.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_AssetManagementInformation" UniqueId="c5dc86c2-b796-6095-8f2a-f131682d1955" ApplicableType="IfcDistributionElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
I would suggest expanding the 2 Pset_ManufacturerXX property sets to include IfcDistributionElement rather than duplicating the same data in this PSet.